bin/cli.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var minimist = require('minimist');
+var checkDependencies = require('../');
+
+var argv = minimist(process.argv.slice(2));
+
+var Cli = {
+  reporter: function (result) {
+    result.error.forEach(function (dep) {
+      console.log(dep);
```

Comment:
Why `console.log` and not `console.error`? Also, it'd be better to change the name `dep` to `msg` or sth like that; after all, these are messages you're logging, not dependencies (i.e. not just dependencies' names are logged but complete messages).